### ISSUE TITLE
bug(menu): does not focus first item when opening with iOS VoiceOver

### ISSUE THREAD

**Issue by zarend** [angular/components#24735]
### Description

When opening the menu with iOS VoiceOver, the first item of the menu should be focused on opening. iOS VoiceOver seems not to select the first item, and linearly navigating forward skips passed the menu completely.

This reproduce on both the material menu and mdc menu.

### Reproduction

Steps to reproduce:
1. on an iOS device with VoiceOver enabled, navigate to [material.angular.io/menu](https://material.angular.io/components/menu/overview)
2. Navigate to the "Basic menu" example.
3. Focus the "Menu" button
4. Activate the button

### Expected Behavior

Menu opens and "Item 1" is selected. VoiceOver announces "Item 1"

### Actual Behavior

Focus moves to the paragraph after the "Basic menu" example. VoiceOver announces "By itself, the"

#### Work-arounds
Tap directly on the menu to give it focus.

Alternative is to continue to linear navigate forward past the end of the page until the menu is focused

### Environment

 - iOS 15.4
 - iPhone SE model MP9D2LL/A
 - Safari Mobile
